src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -151,7 +151,8 @@ const SkillsQuizStepper = () => {
                 {(goal !== DROPDOWN_OPTION_CHANGE_ROLE && (jobs?.length > 0)) ? <SelectJobCard /> : null}
               </div>
               <div>
-                { selectedJob && <SearchCourseCard index={courseIndex} /> }
+                { (selectedJob || skills || goal === DROPDOWN_OPTION_CHANGE_ROLE)
```

Comment:
I think this check will not work as jobs should not be accounted for if `goal === DROPDOWN_OPTION_CHANGE_ROLE`

Reply:
That is already being handled in the custom hook that I wrote. It only extracts skills from job when `goal value !== DROPDOWN_OPTION_CHANGE_ROLE`

```javaScript
if (selectedJob && goal !== DROPDOWN_OPTION_CHANGE_ROLE) {
        interestedJobs.forEach((job) => {
          if (job.name === selectedJob) {
            skillsFromJob = job.skills?.map(skill => skill.name);
          }
        });
}
```

Reply:
Maybe we could discard `goal === DROPDOWN_OPTION_CHANGE_ROLE` because your changes will now ensure that user is not allowed to move to step 2 if no skills are selected, right? so either we have a selectedJob or skills, then the course card component should render. Open to other suggestions too.